.gitignore

```diff
@@ -42,6 +42,12 @@
 # *.iml
 # *.ipr
 
+#Yarn
+yarn.lock
+*.log
+
+#Lock-files
+package-lock.json
```

Comment:
Don't ignore and lock files, since this is an app and lock files are useful for production deployments.